test/source/mock/fes/shared-tenant-fes-endpoints.ts

```diff
@@ -125,3 +127,8 @@ const authenticate = (req: IncomingMessage, type: 'oidc' | 'fes'): string => {
   }
   return MockJwt.parseEmail(jwt);
 };
+
+const messageIdRegex = (req: IncomingMessage) => {
+  const port = parsePort(req);
+  return new RegExp(`{"emailGatewayMessageId":"<(.+)@standardsubdomainfes.localhost:${port}>"}`);
+};
```

Comment:
Maybe you can re-use messageIdRegex from customer-url-fes-endpoint.

Reply:
Good point, implemented it in another PR - https://github.com/FlowCrypt/flowcrypt-browser/pull/4932